common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1420,25 +1498,47 @@ object RssConf extends Logging {
       .toSequence
       .createWithDefaultString("readwrite,diskusage")
 
-  def deviceMonitorEnabled(conf: RssConf): Boolean = conf.get(WORKER_DEVICE_MONITOR_ENABLED)
-
-  def deviceMonitorCheckList(conf: RssConf): Seq[String] = conf.get(WORKER_DEVICE_MONITOR_CHECKLIST)
+  val WORKER_DISK_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.deviceMonitor.check.interval")
+      .withAlternative("rss.disk.check.interval")
+      .categories("worker")
+      .doc("Intervals between device monitor to check disk.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("60s")
 
-  def diskCheckIntervalMs(conf: RssConf): Long = {
-    conf.getTimeAsMs("rss.disk.check.interval", "60s")
-  }
+  val WORKER_DEVICEMONITOR_SYS_BLOCKDIR: ConfigEntry[String] =
+    buildConf("celeborn.worker.deviceMonitor.sys.block.dir")
+      .withAlternative("rss.sys.block.dir")
+      .categories("worker")
+      .doc("The directory where linux file block information is stored.")
+      .stringConf
+      .createWithDefault("/sys/block")
 
-  def slowFlushIntervalMs(conf: RssConf): Long = {
-    conf.getTimeAsMs("rss.slow.flush.interval", "10s")
-  }
+  val WORKER_WRITER_CREATE_MAXATTEMPTS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.writer.create.maxAttempts")
+      .withAlternative("rss.create.file.writer.retry.count")
+      .categories("worker")
+      .doc("Retry count for a file writer to create if its creation was failed.")
+      .intConf
+      .createWithDefault(3)
 
-  def sysBlockDir(conf: RssConf): String = {
-    conf.get("rss.sys.block.dir", "/sys/block")
-  }
+  val WORKER_BASE_DIR_PREFIX: ConfigEntry[String] =
+    buildConf("celeborn.worker.storage.base.dir.prefix")
+      .withAlternative("rss.worker.base.dir.prefix")
+      .categories("worker")
+      .doc("Base directory for Celeborn worker to write if \"base.dir\" is not set.")
+      .stringConf
+      .createWithDefault("/mnt/disk")
 
-  def createFileWriterRetryCount(conf: RssConf): Int = {
-    conf.getInt("rss.create.file.writer.retry.count", 3)
-  }
+  val WORKER_BASE_DIR_COUNT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.storage.base.dir.number")
+      .withAlternative("rss.worker.base.dir.number")
+      .categories("worker")
+      .doc("How many directories will be create if \"base.dir\" is not set." +
```

Comment:
please add a space at the end, and use back quote instead of double quote.

Reply:
Suppose we have the following configurations
```
celeborn.worker.storage.base.dir.prefix=/mnt/disk
celeborn.worker.storage.base.dir.number=3
```
what's the final result? 

1. `/mnt/disk0,/mnt/disk1,/mnt/disk2`
2. `/mnt/disk/0,/mnt/disk/1,/mnt/disk/2`

From my understanding of the doc, the result should be 2, since it says

> How many directories will be create ...

Reply:
The result will be 1.

Reply:
Docs has changed.